src/IonParserTextRaw.ts

```diff
@@ -1049,10 +1049,12 @@ export class ParserTextRaw {
             case T_STRING2:
             case T_STRING3:
                 for (index = this._start; index < this._end; index++) {
+                    let bs = false;
```

Comment:
bs is set to false before each character is read.

Reply:
For readability, consider naming `bs` something like `isEscaped` (or, at least, `backslash`).